Makefile

```diff
@@ -1,4 +1,5 @@
 MAKEFLAGS += --silent
+OVERRIDING_START=foreground
```

Comment:
Probably need to be `OVERRIDING_START ?=` (with `?`) so it’s overridable across many makes (gmake/BSDmake)

Reply:
```suggestion
OVERRIDING_START ?= foreground
```
this @sirn ?

Reply:
Yeap